packages/beacon-node/src/sync/backfill/backfill.ts

```diff
@@ -581,7 +581,7 @@ export class BackfillSync extends (EventEmitter as {new (): BackfillSyncEmitter}
             // we only need to check if prevFinalizedCheckpointBlock is in db
             const prevBackfillCpBlock = await this.db.blockArchive.getByRoot(this.prevFinalizedCheckpointBlock.root);
             if (
-              prevBackfillCpBlock != null &&
+              prevBackfillCpBlock !== null &&
```

Comment:
type of `prevBackfillCpBlock` is ` allForks.SignedBeaconBlock | null`. Same observation [here](https://github.com/ChainSafe/lodestar/pull/4647/files#r996684857) applies